setup/install_gatling/30_classification/20_classify_nodes_via_nc.rb

```diff
@@ -1,13 +1,15 @@
 require 'scooter'
 
 test_name "Classify PE agents via Node Classifier"
+  skip_test 'Installing FOSS, not PE' unless ENV['BEAKER_INSTALL_TYPE'] == 'pe'
```

Comment:
this is no longer case-insenstive.  that might be annoying.  but i'm hoping this code will go away soon?